packages/html/src/grid/tests/grid-filter-toolbar-tool-adaptive-small.tsx

```diff
@@ -0,0 +1,413 @@
+import { Grid, GridToolbar, GridHeader, GridHeaderCell, GridHeaderTable, GridContainer, GridContent, GridTable, GridPager } from "../";
+import { TableThead, TableTbody, TableRow, TableTd } from "../../table";
+import { Button } from "../../button";
+import { ActionSheetFooter, ActionSheetHeader, ActionSheetNormal, Badge, ButtonGroup, ColumnMenuExpander, ColumnMenuItemWrapper, ColumnMenuNormal, DropdownList, FilterMenu, NumericTextbox } from "../..";
+
+const styles = `
+    #test-area {
+        max-width: 780px;
+        --kendo-actionsheet-height: 400px;
+        --kendo-actionsheet-max-height: 400px;
+    }
+    #test-area > section {
+        height: 600px;
+        outline: 1px dotted;
+        overflow: hidden;
+        position: relative;
+        transform: translateZ(0);
+    }
+`;
+
+export default () => (
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-2">
+            <span className="k-col-span-2" >resolutions 500 &lt; 768</span>
+
+            <section>
+                <Grid
+                    className="k-grid-no-scrollbar"
+                    toolbar={
+                        <>
+                            <GridToolbar>
+                                <Button size="large" icon="filter" />
```

Comment:
In my opinion we should set _selected_ state to the Filter button, as the Filter menu is opened only in this case